debezium-connector-postgres/src/main/java/io/debezium/connector/postgresql/PostgresValueConverter.java

```diff
@@ -572,4 +616,16 @@ private boolean isVariableScaleDecimal(final Column column) {
         return (column.scale() == 0 && column.length() == VARIABLE_SCALE_DECIMAL_LENGTH)
                 || (column.scale() == -1 && column.length() == -1);
     }
+
+    public static DebeziumDecimal toSpecialValue(String value) {
```

Comment:
This benefits nicely from returning `Optional` as it allows for `orElse()` at the call site. I'll change it into that.